src/store/indexeddb-remote-worker.js

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
This file is called `src/store/indexeddb-remote-worker.js` and yet defines a single class called `IndexedDbStoreWorker`. Can we please keep them consistent and call it `indexeddb-store-worker`?